### DISCUSSION TITLE
About stun Custom address

### DISCUSSION THREAD

**Discussion by congkuan** [ant-media/Ant-Media-Server#5586] [FAQs]
In addition to stun.wavelength.antmedia.cloud, can I use other custom stun service address?
![image](https://github.com/ant-media/Ant-Media-Server/assets/15060448/43a9bf99-7e31-4bd1-b265-bddc021a925c)

**Comment by muratugureminoglu** [comment]
Hi @congkuan , this STUN/TURN server is serving the AWS Wavelength service and is not accessible from outside. Unfortunately, we do not have a public STUN/TURN server service.
